extensions/ql-vscode/src/databases/config/db-config-store.ts

```diff
@@ -94,7 +94,7 @@ export class DbConfigStore extends DisposableObject {
       );
     }
 
-    const config: DbConfig = cloneDbConfig(this.config);
```

Comment:
Just to check my understanding: Why remove this? Is it because the return type of `cloneDbConfig` is guaranteed to be `DbConfig` anyway?

Reply:
Yep, so it's just unnecessary to set the type - it's inferred from the return type of the function.